src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -67,16 +66,16 @@ public function getImageAction(Request $request)
             $version = (int) (\explode('-', $version)[0] ?? '1');
 
             $mediaProperties = $this->formatCache->analyzedMediaUrl($url);
-
-            return $this->formatManager->returnImage(
-                $mediaProperties['id'],
-                $mediaProperties['format'],
-                $mediaProperties['fileName'],
-                $version,
-            );
-        } catch (ImageProxyException $e) {
+        } catch (ImageProxyUrlNotFoundException $e) {
             throw new NotFoundHttpException('Image create error. Code: ' . $e->getCode(), $e);
         }
+
+        return $this->formatManager->returnImage(
+            $mediaProperties['id'],
+            $mediaProperties['format'],
+            $mediaProperties['fileName'],
+            $version,
+        );
```

Comment:
moved outside as returnImage does catch its own exception and return always a response. So the above catch is only for `ImageProxyUrlNotFoundException` of the `analyzedMediaUrl` used.